test/README.md

```diff
@@ -0,0 +1,19 @@
+Linuxkit Tests Labels
+=====================
+
+## Usage of Artifacts vs Temporary Directory
+
+As the Build Machines have no secrets they will not be able to test running the build output on any cloud providers.
+In this instance, the build tests should copy their output to the `LINUXKIT_ARTIFACTS_DIRECTORY`
+
+## Labels
+
+The `gcp` label is applied when the system where the tests are being run meets the requirements for using Google Cloud Platform.
```

Comment:
it's never set right now as LinuxKitCI has it's own means of running the GCP tests.
the plan is to replace that with a call to `rtf` with the appropriate labels set